### PR TITLE
geda: refdes_renum called /usr/bin/perl

### PR DESCRIPTION
**Pull request by mogorman**
just a small patchphase fix

### CODE REVIEW

File: pkgs/applications/science/electronics/geda/default.nix [hunk 1]
```diff
@@ -12,6 +12,10 @@ stdenv.mkDerivation rec {
   configureFlags = "--disable-update-xdg-database";
   buildInputs = [ pkgconfig guile gtk flex gawk ];
 
+  patchPhase = ''
+    sed -i -e 's/\#!\/usr\/bin\/perl -w/\#!\/usr\/bin\/env perl\nuse warnings;\n;/' ./utils/scripts/refdes_renum
```
Comment (pSub) [inline pkgs/applications/science/electronics/geda/default.nix]: `patchShebangs` seems to be the cleaner solution.

Comment (mogorman) [inline pkgs/applications/science/electronics/geda/default.nix]: i can make that change, but patch-shebangs.sh wont pass on the warning line i take it?  so i will still have to do sed to turn back on the use warning flag right?

**Comment by mogorman** [comment]
can someone tell me what i need to change to get this in?

**Comment by bjornfor** [comment]
Add perl to buildInputs and patchShebangs should be automatically called by the standard builder.

**Comment by mogorman** [comment]
fixed like suggested

**Pull request merged**